### PR TITLE
update managedclusterset

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

File: pkg/hub/submarineragent/controller.go [hunk 1]
```diff
@@ -91,6 +90,7 @@ func NewSubmarinerAgentController(
 		}, clusterInformer.Informer()).
 		WithInformers(clusterSetInformer.Informer()).
 		WithInformers(manifestWorkInformer.Informer()).
+		WithInformers(clusterInformer.Informer()).
```
Comment (skeeey) [inline pkg/hub/submarineragent/controller.go:93]: we already have this informer, this may be duplicated

Comment (qiujian16) [inline pkg/hub/submarineragent/controller.go:93]: is this correct? we already have a clusterinformer

Comment (zhiweiyin318) [reply pkg/hub/submarineragent/controller.go]: deleted

Comment (zhiweiyin318) [reply pkg/hub/submarineragent/controller.go]: deleted

**Pull request merged**